agent/src/beerocks/slave/ap_manager_thread.h

```diff
@@ -81,6 +81,9 @@ class ap_manager_thread : public beerocks::socket_thread {
     void send_heartbeat();
     void send_steering_return_status(beerocks_message::eActionOp_APMANAGER ActionOp,
                                      int32_t status);
+    void remove_client_from_disallowed_list(
+        std::vector<son::ap_manager_thread::disallowed_client_t> &disallowed_clients,
```

Comment:
you are right , i removed the vector.
Although, bssid parameter is always needed since there could be multiple VAPs on this radio that the client could be blocked from.